src/server/controllers/usersController.js

```diff
@@ -20,138 +20,139 @@ const createUser = asyncHandler(async (req, res) => {
             .json({ message: "A user with this email already exists." });
     }
 
-  // Hashing passwords to encrypt user data
-  //const hashPwd = await bcrypt.hash(password,10)
-  const userDocument = { firstname, lastname, email, password };
-  const newUser = await User.create(userDocument);
-  
-  if (newUser) {
-    res.status(201).json({ message: "User successfully created!", id: newUser._id });
-  } else {
-    res.status(400).json({ message: "User unsuccessfully created." });
-  }
-});
+    // Hashing passwords to encrypt user data
+    const hashPwd = await bcrypt.hash(password, 10);
+    const userDocument = { firstname, lastname, email, password: hashPwd };
+    const newUser = await User.create(userDocument);
 
-const getUserByEmail = async (req, res) => {
-  const user = await User.findOne({ email: req.body.email }).then((user) => {
-    if (user) {
-      console.log(`Found user ${user.email}`);
-      res.status(200).json(user);
+    if (newUser) {
+        res.status(201).json({ message: "User successfully created!", id: newUser._id });
     } else {
-      return res.status(400).json({ message: "No user found" });
+        res.status(400).json({ message: "User unsuccessfully created." });
```

Comment:
Wrong status. Should be 201.

Reply:
Isn't 201 for successful creation and 400 for bad request?